src/main/java/org/jboss/aesh/io/FileResource.java

```diff
@@ -104,6 +105,12 @@ public boolean delete() {
     }
 
     @Override
+    public void move(Resource target) throws IOException {
+        FileResource destination = (FileResource) target;
+        Files.move(this.file.toPath(), destination.file.toPath(), StandardCopyOption.REPLACE_EXISTING);
```

Comment:
thats a good input soro, im not sure whats best tbh. im fine with both.
i think ill accept the pr since we're doing the same thing for copy/write etc. we there is no extra flag if you overwrite an existing file there